Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewViewModel.kt

```diff
@@ -63,13 +65,15 @@ class PersonOverviewViewModel @AssistedInject constructor(
     ) {
         persons.filterNotPending()
             .forEachIndexed { index, person ->
-                val admissionState = person.admissionState
+                val admissionState = person.dccWalletInfo?.admissionState
+                val certificatesForOverviewScreen = person.certificatesForOverviewScreen
+                Timber.d("VerificationCertificates ${person.certificatesForOverviewScreen}")
                 val color = PersonColorShade.shadeFor(index)
-                if (admissionState != null) {
+                if (certificatesForOverviewScreen.isNotEmpty()) {
```

Comment:
Please check the ViewModel Tests? which is not currently disabled

Reply:
View model tests pass after enabling the disabled tests.